src/vmm/src/builder.rs

```diff
@@ -1088,6 +1088,15 @@ pub mod tests {
         .unwrap()
     }
 
+    fn cmdline_contains(cmdline: &Cmdline, slug: &str) -> bool {
+        cmdline
+            .as_cstring()
+            .unwrap()
+            .into_string()
+            .unwrap()
+            .contains(slug)
+    }
+
```

Comment:
We can remove the `unwrap()`s so if they where `Err` it would panic on the specific assertion line rather than in this function, giving more information on what broke.
```suggestion
    fn cmdline_contains(cmdline: &Cmdline, slug: &str) -> bool {
        if let Ok(cstring) = cmdline.as_cstring() {
            if let Ok(string) = cstring.into_string() {
                if string.contains(slig) {
                    return true;
                }
            }
        }
        false
    }
```

Reply:
Yeah, I agree with @roypat . Closing this comment to unblock the PR merge.

Reply:
In the case an unwrap will never fail it should be commented.

Reply:
I will open a new PR for adding the comment as suggested! Sorry for hurry-up with the PR, it wasn't intentional.